src/App.tsx

```diff
@@ -1,10 +1,30 @@
+import GlobalStyle from "globalStyles";
+import styled from "styled-components";
+// import { CreateAccount } from "pages/CreateAccount";
+import { HomeLayout } from "pages/HomeLayout";
+// import { Login } from "pages/Login";
```

Comment:
```suggestion
import styled from "styled-components";

import GlobalStyle from "globalStyles";
// import { CreateAccount } from "pages/CreateAccount";
import { HomeLayout } from "pages/HomeLayout";
// import { Login } from "pages/Login";
```

It's good to separate imports from external libraries and imports from the app itself using a blank line
Also kudos for proper alphabetization

Reply:
Thanks! Did this.